backend/src/routes/chat_route.py

```
@@ -0,0 +1,29 @@
+    db=SessionLocal()
+    try:
+        yield db #iterator hai
+    finally:
+        db.close()
+
+
+@chatRouter.post("/chat")
+async def ChatResult(user_request:schema.ChatInput,db:Session=Depends(get_db),user:dict=Depends(getCurrentUser)):
+    print("Request By:",user)
+    result = await run(user_request)
+    output=result
```

Ruff (0.12.2)

Local variable `output` is assigned to but never used

Remove assignment to unused variable `output`

(F841)